ml-agents/mlagents/trainers/components/reward_signals/gail/model.py

```diff
@@ -261,5 +292,8 @@ def create_loss(self, learning_rate: float) -> None:
             )
         else:
             self.loss = self.discriminator_loss
+
+        self.loss = self.loss + self.gradient_penalty * self.compute_gradient_penalty()
```

Comment:
Maybe don't use `self.compute_gradient_penalty()` if `self.gradient_penalty == 0.0` (to allow turning it off if you really don't want it for some reason)?

Reply:
Made it only happen if the penalty is greater than 0